builds/fabric8-che/plugins/che-plugin-bayesian-lang-server/src/main/java/org/eclipse/che/plugin/languageserver/bayesian/server/launcher/BayesianLanguageServerLauncher.java

```diff
@@ -87,8 +87,10 @@ private static LanguageServerDescription createServerDescription() {
                                                                               null),//
                                                            new DocumentFilter(JsonModule.LANGUAGE_ID,//
                                                                               "package\\.json",//
-                                                                              null)//
-                                             // TODO pom.xml
+                                                                              null),
+                                                           new DocumentFilter("pom",
```

Comment:
If such a constant had been available in the maven language server I wold have taken it, but unfortunately the related code in the maven language server launcher is the following:

```java
    @Override
    public LanguageServerDescription getDescription() {
        return new LanguageServerDescription("org.eclipse.che.plugin.maven", Collections.singletonList("pom"), Collections.emptyList());
    }
```

On the other hand `pom` is quite and expected string for a pom file ;-)